force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -340,10 +492,50 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Person createPerson(String name, Integer age) {
         // Create a new instance of the Person class using the provided name and age
+        public class Person
+        {
+            private String name;
+            private Integer age;
+
+            public Person(String name, Integer age)
+            {
+                this.name = name;
+                this.age = age;
+            }
+            public String getName() 
+            {
+                return name;
+            }
+        
+            public Integer getAge() 
+            {
+                return age;
+            }
+        
+            public void setName(String name) 
+            {
+                this.name = name;
+            }
+        
+            public void setAge(Integer age) 
+            
+                this.age = age;
+            }
+        }
+        
+        public class PersonService {
+            public static Person createPerson(String name, Integer age) 
+            {
+                Person newPerson = new Person(name, age);
+                return newPerson;
+            }
+        }
```

Comment:
Here you shouldn't be recreating the Person class that already exists, instead  you simply take advantage of it and call it:
`Person newPerson = new Person(name, age);`